app/assets/stylesheets/common/base/menu-panel.scss

```diff
@@ -208,6 +208,20 @@
   #quick-access-profile {
     display: inline;
 
+    .set-user-status {
+      .emoji {
+        padding-top: 0;
+      }
+
+      .relative-date {
+        text-align: left;
+        font-size: var(--font-down-3);
+        padding-top: 0.25em;
+        margin-left: 0.75em;
+        color: var(--primary-medium);
+      }
```

Comment:
We do have some CSS for `.relative-date` at line 236 for the Do Not Disturb button, maybe we can reuse some of that CSS?

Reply:
We can for sure, I did that.